File: Ent_UsAdmin/Cajas.vb
Imports System.Data.SqlClient

Public Class Cajas
    Private Sub Cajas_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        cargarCajas()
    End Sub

    Public Sub cargarCajas()
        dtimpuestos.Rows.Clear()
        Try
            Dim strimpuestos As String
            iniciarconexionempresa()

            strimpuestos = "select * from CajasSucursal"

            Dim ejec = New SqlCommand(strimpuestos)
            ejec.Connection = conexionempresa
            Dim id, numero, valor, factor, tipo As String

            Dim myreaderimpuestos As SqlDataReader = ejec.ExecuteReader()
            While myreaderimpuestos.Read
                id = myreaderimpuestos("id")
                numero = myreaderimpuestos("nocaja")

                dtimpuestos.Rows.Add(id, numero, FormatCurrency(myreaderimpuestos("Fondo"), 2), FormatCurrency(myreaderimpuestos("limite"), 2))
            End While
            myreaderimpuestos.Close()
        Catch ex As Exception
            MessageBox.Show(ex.Message)
        End Try

    End Sub

    Private Sub btn_agregar_Click(sender As Object, e As EventArgs) Handles btn_agregar.Click
        AgregarCaja.Show()
    End Sub

    Private Sub BunifuThinButton21_Click(sender As Object, e As EventArgs) Handles BunifuThinButton21.Click
        cargarCajas()
    End Sub
End Class

File: Ent_UsAdmin/Caja/Cajas.cs
using System;
using System.Data.SqlClient;
using System.Windows.Forms;
using Microsoft.VisualBasic;
using Microsoft.VisualBasic.CompilerServices;

namespace ConfiaAdmin
{

    public partial class Cajas
    {
        public Cajas()
        {
            InitializeComponent();
        }
        private void Cajas_Load(object sender, EventArgs e)
        {
            cargarCajas();
        }

        public void cargarCajas()
        {
            dtimpuestos.Rows.Clear();
            try
            {
                string strimpuestos;
                Module1.iniciarconexionempresa();

                strimpuestos = "select * from CajasSucursal";

                var ejec = new SqlCommand(strimpuestos);
                ejec.Connection = Module1.conexionempresa;
                string id, numero, valor, factor, tipo;

                var myreaderimpuestos = ejec.ExecuteReader();
                while (myreaderimpuestos.Read())
                {
                    id = Conversions.ToString(myreaderimpuestos["id"]);
                    numero = Conversions.ToString(myreaderimpuestos["nocaja"]);

                    dtimpuestos.Rows.Add(id, numero, Strings.FormatCurrency(myreaderimpuestos["Fondo"], 2), Strings.FormatCurrency(myreaderimpuestos["limite"], 2));
                }
                myreaderimpuestos.Close();
            }
            catch (Exception ex)
            {
                MessageBox.Show(ex.Message);
            }

        }

        private void btn_agregar_Click(object sender, EventArgs e)
        {
            My.MyProject.Forms.AgregarCaja.Show();
        }

        private void BunifuThinButton21_Click(object sender, EventArgs e)
        {
            cargarCajas();
        }
    }
}